source/upgrade/manager.rst

```diff
@@ -57,64 +60,54 @@ Review the changes made to the configuration repository and commit the changes:
 
 The directories ``environments/manager/roles`` and
 ``environments/manager/.venv`` need to be deleted on the manager node.
+They will be recreated and populated automatically.
 
 .. code-block:: console
 
    rm -rf /opt/configuration/environments/manager/roles
    rm -rf /opt/configuration/environments/manager/.venv
 
-After updating the configuration repository, the manager is now updated.
+Upgrading to Manager version 2019.4.0
+=====================================
```

Comment:
--------- instead of ========= (it a sub section of the manager section)

Reply:
Fixed.